Makefile

```
@@ -93,6 +93,15 @@ MINIKUBE_PROFILE ?= minikube
 KIND ?= $(shell which kind 2>/dev/null || echo "MISSING-KIND-FROM-PATH")
 KIND_NAME ?= kind
 
+# Determine if the OC is operational or not. Useful for other commands that might timeout if the OC exists but is not responding.
+ifeq ($(CLUSTER_TYPE),minikube)
+OC_READY ?= $(shell if minikube -p ${MINIKUBE_PROFILE} status &>/dev/null ; then echo "true" ; else echo "false" ; fi)
```

We have a ${MINIKUBE} variable for this - https://github.com/kiali/kiali/blob/master/Makefile#L89

In some contexts you can't rely it being on path - hence, some times we need to pass in `MINIKUBE=/full/path/here`

```suggestion
OC_READY ?= $(shell if ${MINIKUBE} -p ${MINIKUBE_PROFILE} status &>/dev/null ; then echo "true" ; else echo "false" ; fi)
```